### DISCUSSION TITLE
機器學習課程

### DISCUSSION THREAD

**Discussion by timchen1216** [1101-GS3073-C/Python-Community-GS3073-C#260] [General]
台大李弘毅老師的機器學習課程，老師幽默風趣，講解課程深入淺出，歡迎想要了解機器學習的人觀看。
課程：
https://youtube.com/playlist?list=PLJV_el3uVTsMhtt7_Y6sgTHGHp1Vb2P2J
作業及上課投影片：
https://speech.ee.ntu.edu.tw/~hylee/ml/2021-spring.php

**Comment by EricAgumonJou** [comment]
我來分享一下李宏毅老師機器學習課程的筆記：https://hackmd.io/@shaoeChen/B1CoXxvmm/https%3A%2F%2Fhackmd.io%2Fs%2FHyKhr5sRz
